qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -61,29 +61,44 @@
+            RHEL.8, RHEL.9.0, RHEL.9.1, RHEL.9.2, RHEL.9.3, RHEL.9.4:
+                flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
             model_pattern = "AMD EPYC-Rome Processor%s"
             RHEL.6, RHEL.7:
                 flags = ""
         - EPYC-Rome-v2:
             only HostCpuVendor.amd
-            flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
+            flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip"
+            RHEL.8, RHEL.9.0, RHEL.9.1, RHEL.9.2, RHEL.9.3, RHEL.9.4:
+                flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
             model_pattern = "AMD EPYC-Rome Processor%s"
             RHEL.6, RHEL.7:
                 flags = ""
         - EPYC-Rome-v3:
             only HostCpuVendor.amd
             required_qemu = [8.0.0,)
-            flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
+            flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip"
+            RHEL.8, RHEL.9.0, RHEL.9.1, RHEL.9.2, RHEL.9.3, RHEL.9.4:
+                flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip xsaves"
             model_pattern = "AMD EPYC-Rome-v3 Processor%s"
             RHEL.6, RHEL.7:
                 flags = ""
+        - EPYC-Rome-v4:
+            only HostCpuVendor.amd
+            required_qemu = [8.0.0,)
+            flags = "movbe rdrand rdtscp fxsr_opt cr8_legacy osvw fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 arat f16c xsaveerptr clzero rdpid perfctr_core ibpb wbnoinvd stibp clwb umip"
+            model_pattern = "AMD EPYC-Rome-v4 Processor%s"
+            RHEL.6, RHEL.7:
+                flags = "
```

Comment:
typo?

Reply:
> typo?

No, we support AMD Zen3 since RHEL 8, this is kind of negative test.

Reply:
I mean the single `"`

Reply:
Correct it, thanks!